lib/heroku/kensa/check.rb

```diff
@@ -173,6 +173,14 @@ def call!
           response.is_a?(Hash) && response.has_key?("id")
         end
 
+        check "id does not contain heroku_id" do
+          if response["id"] && response["id"].respond_to?(:include?) && response["id"].include?(data["heroku_id"])
```

Comment:
`reponse["id"]` is mandatory (if this isn't tested, we should add that too), so it should always be here and non-blank.

It should also always be a string, too. However, for legacy reasons there may be some who sent us ints, so we coerce it to string in the back end. I'd just coerce `to_s` then check if it includes the ID.

Lastly, we don't want to check if it just includes `heroku_id`. Some strip it or regex out the number from it. We should check if the numeric ID portion is present, IMO